src/twisted/mail/test/test_imap.py

```diff
@@ -1161,6 +1161,24 @@ def test_fetchParserBody(self):
             b"BODY[1.3.9.11.HEADER.FIELDS.NOT (Message-Id Date)]<103.69>",
         )
 
+    def test_fetchParserParseStingPartial(self):
+        """
+        Partial requests starting at 0 and longer than the message
+        do not return a partial response.
```

Comment:
Thanks for adding the docstring...

just asking.. (at 00:18 AM): What is a partial response ?

Reply:
Ah, that is badly worded. I have reworded it to make a bit more sense, I hope.